src/db/build.ts

```diff
@@ -1,9 +1,17 @@
 import { sequelize, Event, User } from '.'
 import fakeData from './FakeData/fakeData.json'
+
+const { NODE_ENV } = process.env
```

Comment:
why you are using `NODE_ENV` from `proccess.env` ?

it's already in the `config/environment.ts` file